airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/spec.py

```diff
@@ -65,6 +65,7 @@ class Config:
         description="The date from which you'd like to replicate data for this stream, in the format YYYY-MM-DDT00:00:00Z.",
         pattern=DATE_TIME_PATTERN,
         examples=["2017-01-25T00:00:00Z"],
+        format="date-time"
```

Comment:
We don't need to add this here because pydantic add it automatically if type of field is `datetime`

`start_date: Optional[datetime] = Field(`

Reply:
Updated